test/validateOptions.js

```diff
@@ -13,14 +13,14 @@ describe('Plugin option validation', function() {
     });
     describe('Given two options', function(){
        it('Should return an error', function(){
-           plugin.register(null, {lower_case: true, camel_case: true}, function(err) {
+           plugin.register(null, {lowerCase: true, camelCase: true}, function(err) {
                 expect(err.toString()).to.equal('ValidationError: child "camel_case" fails because ["camel_case" contains an invalid value]. child "lower_case" fails because ["lower_case" contains an invalid value]. child "all_caps" fails because ["all_caps" is required]. child "first_cap" fails because ["first_cap" is required]');
             });
        });
     });
     describe('Given all options', function(){
        it('Should return an error', function(){
-           plugin.register(null, {lower_case: true, camel_case: true, all_caps: true, first_cap: true }, function(err) {
+           plugin.register(null, {lowerCase: true, camelCase: true, allCaps: true, firstCap: true }, function(err) {
                expect(err.toString()).to.equal('ValidationError: child "camel_case" fails because ["camel_case" contains an invalid value]. child "lower_case" fails because ["lower_case" contains an invalid value]. child "lower_case" fails because ["lower_case" contains an invalid value]. child "lower_case" fails because ["lower_case" contains an invalid value]');
```

Comment:
these validation messages look incorrect. Have you run these tests?

Reply:
yes i have